Client/Frontend/Widgets/ThumbnailCell.swift

```diff
@@ -27,8 +32,10 @@ class ThumbnailCell: UICollectionViewCell {
         imageView.layer.borderColor = BorderColor.CGColor
         imageView.layer.borderWidth = 2
         imageView.layer.cornerRadius = 3
+        imageView.clipsToBounds = true
         imageView.snp_makeConstraints({ make in
             make.top.left.right.equalTo(self.contentView).insets(CellInsets)
+            make.width.equalTo(self.imageView.snp_height).multipliedBy(ThumbnailCellUX.ImageAspectRatio)
```

Comment:
It's setting the aspect ratio. So this line isn't setting the width, but rather setting a constraint to keep the width and height to the correct proportions.